### PR TITLE
Add timeout

### PR DESCRIPTION
**Pull request by exacs**

### CODE REVIEW

**Comment by me97esn** [comment]
No, we have to handle the timeouts somehow. I tested by setting the timeout to 1ms, then the entire process crashes on first message. We should at least catch the timeout error, log it in the standard way and prevent the app from crashing

File: messages/handleUserMessage.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ async function createOrUpdate (user) {
     })
     await canvasApi.requestCanvas(`accounts/1/logins/${login.id}`, 'PUT', user)
   } catch (e) {
-    if (e.statusCode === 404) {
+    if (e.response.statusCode === 404) {
```
Comment (me97esn) [inline messages/handleUserMessage.js:83]: What's the reason for this change? Are the results from CanvasApi different in this version?

Comment (exacs) [reply messages/handleUserMessage.js:83]: Exactly. Before we were using `got v9` and now we are using `got v11`. Internally `@kth/canvas-api` is just returning whatever is returned by `got`

Comment (me97esn) [reply messages/handleUserMessage.js:83]: Ok, makes sense. I just want to know that it's correct.
